### PR TITLE
add search bars

### PR DESCRIPTION
**Pull request by manch0ffline**
- [DEMO LINK](https://manch0ffline.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://manch0ffline.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,6 +11,27 @@
...
+        <input
+          class="main__input main__input--big"
+          type="text"
+          placeholder="Try &#34;Los Angeles&#34;"
+          id="input-big"
+          data-qa="big"
+        >
+      </label>
+      <label for="input-small" class="main__label main__label--small">
+        <div class="img img--small"></div>
```
Comment (loralevitska) [inline src/index.html:26]: Try to use `img` tag for this

File: src/index.html [hunk 2]
```diff
@@ -11,6 +11,27 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <main class="main">
+      <label for="input-big" class="main__label main__label--big">
+        <div class="img img--big"></div>
```
Comment (loralevitska) [inline src/index.html:16]: Try to use img tag for this

**Review by loralevitska — CHANGES_REQUESTED** [review]
Almost perfect) Let's improve

**Review by loralevitska — CHANGES_REQUESTED** [review]
Please redeploy your solution for review 
image
If you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review

File: src/style.css [hunk 3]
```diff
@@ -1 +1,100 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 400;
```
Comment (volodymyr-soltys97) [inline src/style.css:9]: Default value, so you can just remove it
```suggestion
```

**Review by volodymyr-soltys97 — APPROVED** [review]
Great work 🔥

**Pull request closed without merging**